src/internal/mock_pfs/mock.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
It's triggered by `go generate`, and that shell script can even go away once the bug referenced below is fixed.